src/models/ErrorHandler.js

```diff
@@ -0,0 +1,32 @@
+import { Logger } from 'zos-lib'
+
+const log = new Logger('Error')
+
+export default class ErrorHandler {
+  constructor(error, { verbose }) {
+    this.error = error
+    this.verbose = verbose
+  }
+
+  call() {
+    const errorName = this.error.name || 'Undefined'
+    const handlerName = `on${errorName}`
+    this[handlerName]()
```

Comment:
If the name of the error is not "Error", this is going to fail. I'd consider changing this dynamic dispatch to something more simple: if the error has a `message` property, output it; otherwise, go for the "undefined error" approach.